templates/sso/sso_base.html

```diff
@@ -0,0 +1,68 @@
+
+{% load render_bundle from webpack_loader %}
+ {% load sekizai_tags %}
+
+<!DOCTYPE html>
+<html lang="nb-NO">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <title>{% block title %}Online, linjeforeningen for informatikk ved NTNU{% endblock %}</title>
+    <meta name="description" content="Online er linjeforeningen for informatikkstudentene ved NTNU. Våre studenter går Bachelor- eller Mastergradstudium i informatikk.">
+    <link rel="icon" href="{{ STATIC_URL }}img/favicon.png">
+    <link rel="icon" type="image/png" href="{{ STATIC_URL }}mimg/favicon-32x32.png" sizes="32x32">
+    <link rel="icon" type="image/png" href="{{ STATIC_URL }}mimg/favicon-194x194.png" sizes="194x194">
+    <link rel="icon" type="image/png" href="{{ STATIC_URL }}mimg/favicon-96x96.png" sizes="96x96">
+    <link rel="icon" type="image/png" href="{{ STATIC_URL }}mimg/android-chrome-192x192.png" sizes="192x192">
+    <link rel="icon" type="image/png" href="{{ STATIC_URL }}mimg/favicon-16x16.png" sizes="16x16">
+    <meta name="theme-color" content="#005fa2">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    {% block styles %}
+    {% endblock %}
+    {% render_bundle 'common' 'css' %}
+    {% render_bundle 'core' 'css' %}
+    {% block scripts %}
+    {% endblock %}
+    {% render_block "css" %}
+    {% block apple-webapp %}{% endblock %}
+</head>
+<body>
+
+    <div class="container" id="isloading-component"></div>
+    {% block content %}{% endblock %}
+
+    {% block js %}
+        <script
+            src="https://browser.sentry-cdn.com/5.21.1/bundle.min.js"
+            integrity="sha384-uFON5MXfE3NkWO60hM8JaXAYAV0fqTCQbdy282z8YfelqBoXx+F7JFzX6+WfFIML"
+            crossorigin="anonymous">
+        </script>
+        <script>Sentry.init({ dsn: '{{ sentry_dsn }}' })</script>
+        <script
+            src="https://code.jquery.com/jquery-2.2.4.min.js"
+            integrity="sha256-BbhdlvQf/xTY9gja0Dq3HiwQF8LaCRTXxZKRutelT44="
+            crossorigin="anonymous"
+        ></script>
+        <script>
+            window.jQuery || document.write('<script src="{{ STATIC_URL }}js/libs/jquery-2.2.4.min.js"><\/script>');
```

Comment:
@FluidSense this is the line triggering the security-warning from LGTM, the commit it is referring to is just the latest commit in `main`

Reply:
I see, thanks for clarifying! That is just the regular base-html stripped of the nav-bar and footer, I believe I should be able to remove jquery as I am not aware of anything using it on the authorization-page!